src/Control/Eff/Internal.hs

```diff
@@ -22,8 +22,36 @@
 -- Extensible Effects are implemented as typeclass constraints on an Eff[ect] datatype.
 -- A contrived example can be found under "Control.Eff.Example". To run the
 -- effects, consult the tests.
-module Control.Eff.Internal ( module Control.Eff.Internal
-                            ) where
+module Control.Eff.Internal
+  (
+    -- * the Effect type
+    Eff(..)
```

Comment:
@power-fungus i don't like the duplication of listing out functions both in `Control.Eff.Internal` and in the export list of `Control.Eff.ImplementYourOwnEffect`
How can we eliminate that?

Reply:
I don't mind. `Internal` is not an exported module anyway and I thought this increases documentation a bit. I feel it is necessary for increasing code-documentation.

If necessary, it would be possible to export everything from `Internal`, like it was before.

Reply:
@power-fungus i am not sure i understand.

- today:
  - `Internal` has all the base definitions including some which are needed by `Control.Eff.Lift`

Reply:
@suhailshergill Indeed, there is no functional benefit here. I will remove it as the `Extend` module contains a very similar list.

Reply:
:+1: